src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -592,3 +592,22 @@ Tests:
     ThenReturns:
     - { "date" : { "$date" : { "$numberLong" : "1369875424299" } } }
 
+  - Name: Inc0
+    GivenTemplate:
+      int: {^Inc: {multiplier: 100, step: 1}}
+    ThenReturns:
+    - "int": { "$numberLong" : "201"}
```

Comment:
Here you should have multiple entries under ThenRetuns like
`- "int": { "$numberLong" : "201"}`
`- "int": { "$numberLong" : "202"}`
`- "int": { "$numberLong" : "203"}`